base64ct/src/lib.rs

```diff
@@ -391,8 +395,9 @@ fn encode<'a>(
 /// If `input` length is greater than `usize::MAX/4`.
 #[cfg(feature = "alloc")]
 #[cfg_attr(docsrs, doc(cfg(feature = "alloc")))]
+#[inline(always)]
 fn encode_string(input: &[u8], padded: bool, hi_bytes: (u8, u8)) -> String {
-    let elen = encoded_len(input, padded);
+    let elen = encoded_len_inner(input.len(), padded).expect("input is too big");
```

Comment:
Thanks to the forced inlining, the second `expect` used on `encode` result will be properly removed.